src/main/java/com/redislabs/redisai/Model.java

```diff
@@ -314,11 +352,26 @@ public Model setMinBatchTimeout(long minBatchTimeout) {
     }
 
     args.add(Keyword.BLOB.getRaw());
-    args.add(blob);
+    collectChunks(args, blob);
 
     return args;
   }
 
+  private static void collectChunks(List<byte[]> collector, byte[] array) {
+    final int chunkSize = BLOB_CHUNK_SIZE;
```

Comment:
@sazzad16 why do you need this local variable and not use the constant?

Reply:
@gkorland just for code readability. This variable is used several times and a smaller variable name seemed better.

Reply:
I find it more confusing...